src/assets/i18n/pt-BR.json5

```diff
@@ -27,14 +27,14 @@
   "500.help": "O servidor está temporariamente impossibilitado de atender sua solicitação devido a um período de manutenção ou problemas no servidor. Por favor, tente novamente mais tarde.",
 
   // "500.link.home-page": "Take me to the home page",
-  "500.link.home-page": "Leve-me para a página inicial",
+  "500.link.home-page": "Retornar à página inicial",
 
 
   // "404.help": "We can't find the page you're looking for. The page may have been moved or deleted. You can use the button below to get back to the home page. ",
-  "404.help": "Não pudemos encontrar a página pela qual procura. A página pode ter sido movida ou apagada. Você pode utilizar o botão abaixo para voltar a página inicial. ",
+  "404.help": "Não encontramos a página que você procura. A página pode ter sido movida ou apagada. Você pode utilizar o botão abaixo para voltar à página inicial. ",
 
   // "404.link.home-page": "Take me to the home page",
-  "404.link.home-page": "Leve-me a página inicial",
+  "404.link.home-page": "Retornar à página inicial",
```

Comment:
The same as in previous comments... It makes more sense to have:
`Acessar a página inicial`